<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newSWallet</class>
 <widget class="QDialog" name="newSWallet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New SWallet</string>
  </property>
  <widget class="QGroupBox" name="registration_Box">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>481</width>
     <height>411</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="title">
    <string>New SWallet Registration:</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QFrame" name="fullName_frame">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="fullName_lineEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>First Last</string>
     </property>
    </widget>
    <widget class="QLabel" name="fullName_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Full Name</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="password_frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="password_lineEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Pass</string>
     </property>
    </widget>
    <widget class="QLabel" name="password_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="nationalID_frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="nationalID_lineEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="nationalID_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>National ID</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="age_frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="age_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Age</string>
     </property>
    </widget>
    <widget class="QComboBox" name="age_comboBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="currentText">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="phoneNumber_frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="phoneNumber_lineEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>123 1234567 (Last 10 digits)</string>
     </property>
    </widget>
    <widget class="QLabel" name="phoneNumber_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Phone Number  +20</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="create_Button">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>170</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(87, 227, 137);
color: black;</string>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_Button">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>240</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(237, 51, 59);
color: black;</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>phoneNumber_lineEdit</tabstop>
  <tabstop>fullName_lineEdit</tabstop>
  <tabstop>password_lineEdit</tabstop>
  <tabstop>nationalID_lineEdit</tabstop>
  <tabstop>age_comboBox</tabstop>
  <tabstop>create_Button</tabstop>
  <tabstop>cancel_Button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
